tensorflow/compiler/tf2tensorrt/utils/trt_shape_optimization_profiles.h

```diff
@@ -145,7 +168,8 @@ class TrtShapeOptimizationProfile {
 
   // Maps input vector shapes to TRT Optimization profiles (min, max, opt) i.e.
   // maps input_shapes_ to profiles_
```

Comment:
Let's revise this to reflect the new implementation:
Creates optimization profiles profiles_ for the set of concrete input shapes collected in input_shapes_. The input_partial_shapes of the network is used to ensure that the created optimization profiles are compatible with the network.